vm_supervisor/storage.py

```diff
@@ -271,7 +274,10 @@ async def create_devmapper(
     snapshot_table_command = f"0 {extended_block_size} snapshot {path_base_device_name} {extended_loop_device} P 8"
     await create_mapped_device(mapped_volume_name, snapshot_table_command)
 
-    await e2fs_check_and_resize(path_mapped_volume_name)
+    mount_path = Path(f"/mnt/{mapped_volume_name}")
+    if not mount_path.is_dir():
```

Comment:
pathlib has `mount_path.mkdir(parents=True, exist_ok=True)` to avoid having to check this.